src/innsending/api.ts

```diff
@@ -11,3 +12,19 @@ export const sendInnSøknad = (søknad: object) => {
       return response.data;
     });
 };
+
+export const mapBarnTilEntenIdentEllerFødselsdato = (barn: IBarn[]) => {
+  const filtrerteBarn = barn.map((barn) => {
+    if (barn.lagtTil) {
+      if (barn.fødselsdato && !barn.fødselsdato.verdi) {
+        const endretBarn: IBarn = {
+          ...barn,
+          fødselsdato: { ...barn.fødselsdato, verdi: '' },
+        };
```

Comment:
_kan_ hende vi må ha en `delete endretBarn.fødselsdato` for at dette skal fungere mot API-et... :/ 

Eller hva tenker du @ma10s ?